server/app/attestation-service-ctl/src/api.rs

```diff
@@ -0,0 +1,159 @@
+use anyhow::Result;
+use clap::{App, Arg, SubCommand};
+use shadow_rs::shadow;
+use std::path::Path;
+
+mod attestation;
+mod management;
+
+pub mod management_api {
+    tonic::include_proto!("management");
+}
+pub mod attestation_api {
+    tonic::include_proto!("attestation");
+}
+pub mod common {
+    tonic::include_proto!("common");
+}
+
+#[macro_use]
+extern crate log;
+shadow!(build);
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    env_logger::builder()
+        .filter(None, log::LevelFilter::Info)
+        .init();
+
+    let version = format!(
+        "\nv{}\ncommit: {}\nbuildtime: {}",
+        build::PKG_VERSION,
+        build::COMMIT_HASH,
+        build::BUILD_TIME
+    );
+
+    let matches = App::new("client")
+    .version(version.as_str())
+    .long_version(version.as_str())
+    .author("Confidential-Containers Team")
+    .subcommand(SubCommand::with_name("set-policy")
+        .about("Set the <TEE> specific evaluation `Policy(.rego)` according to the content of <POLICY>.")
+        .arg(Arg::from_usage("--addr=[ADDR] 'Attestation Service's `management-sock` address.'"))
+        .arg(Arg::from_usage("--tee=[TEE] 'The target TEE name'"))
```

Comment:
`--addr`  and `--tee`are global parameters that could be processed separately. They're not specific to only a tiny subset of commands.